ansible_base/authentication/models/authenticator.py

```diff
@@ -33,6 +33,18 @@ class Authenticator(UniqueNamedCommonModel):
     slug = fields.SlugField(max_length=1024, default=None, editable=False, unique=True, help_text="An immutable identifier for the authenticator")
     category = fields.CharField(max_length=30, default=None, help_text="The base type of this authenticator")
 
+    auto_migrate_users_to = ForeignKey(
+        "Authenticator",
+        help_text=(
+            "Automatically move users from this authenticator to the target authenticator when a matching user logs in via the target authenticator. "
+            "For this to work, the field used for the user ID on both authenticators needs match up. This should only be used when migrating users "
```

Comment:
```suggestion
            "For this to work, the field used for the user ID on both authenticators needs to match up. This should only be used when migrating users "
```

But I'd maybe consider "needs to be the same" or "needs to have the same value" or something a bit more formal than "match up"